torch/csrc/jit/OVERVIEW.md

```diff
@@ -1295,18 +1298,17 @@ and add an edge from `%output` to `%self`. The alias analysis pass is flow-insen
 As a more involved example, the following TorchScript snippet:
 ```python
 @torch.jit.script
-def foo(a : Tensor, b : Tensor):
-        c = 2 * b
-  a += 1
-  if a.max() > 4:
-    r = a[0]
-  else:
-    r = b[0]
-  return c, r
+def foo(a: Tensor, b: Tensor):
+    c = 2 * b
+    if a.max() > 4:
+        r = a[0]
+    else:
+        r = b[0]
+    return c, r
 ```
 Will produce a graph like this:
 
-![AliasTracker graph](https://github.com/pytorch/pytorch/blob/master/docs/source/_static/img/aliastracker_graph.png)
+![MemoryDAG graph](https://github.com/pytorch/pytorch/blob/master/docs/source/_static/img/MemoryDAG_graph.png)
```

Comment:
Is this `png` checked in? I tried to access `MemoryDAG_graph.png` and it doesn't exist. But `aliastracker_graph.png` does.